app/src/app.rs

```diff
@@ -41,9 +41,10 @@ impl App {
 		Ok(())
 	}
 
-	fn dispatch_quit(&mut self) {
+	fn dispatch_quit(&mut self, write_cwd_file: bool) {
 		if let Some(p) = &BOOT.cwd_file {
-			let cwd = self.cx.manager.cwd().as_os_str();
+			let cwd =
+				if write_cwd_file { self.cx.manager.cwd().as_os_str() } else { &BOOT.cwd.as_os_str() };
```

Comment:
If the sole purpose is to avoid triggering `cd` not creating a `cwd-file` is a better choice compared to writing `cwd` into `cwd-file`, and `ya` wrapper doesn't execute `cd` when it detects the file not exist. Any other reasons?